src/main/app.ts

```diff
@@ -488,6 +494,10 @@ export class KandoApp {
     // fullscreen applications.
     this.window.setAlwaysOnTop(true, 'screen-saver');
 
+    // We set the Activation Policy to Accessory so that Kando doesn't show in dock
+    // or the CMD Tab App Switcher
+    app.setActivationPolicy('accessory')
```

Comment:
I think this is mac exclusive, a windows user needs to test it.

Reply:
Got it. Added a commit with a condition which should make it mac-only.